src/UnitTests/Gui/Windows/ProjectBrowserServiceTests.cs

```diff
@@ -393,12 +387,25 @@ private void Given_ProgramWithOneSegment()
             project.Programs.Add(program);
         }
 
+        private void Given_ImageMapItem()
+        {
```

Comment:
Consider refactoring this to `Given_ImageMapItem(uint address)` so this could be used in tests with multiple globals.